pyquil/quilbase.py

```diff
@@ -31,6 +31,7 @@
     Tuple,
     Union,
     TYPE_CHECKING,
+    Sequence,
```

Comment:
nitpick: this list of imports was sorted alphabetically, and this new entry is out of place.

Reply:
🤣

Reply:
We don't have 'q' in the polish alphabet so You made be check whether Set should be before or after Sequence :)